tests/ctags/simple.d.tags

```diff
@@ -20,5 +20,5 @@ obj
 qar�64�Struct.Union�0�int
 quxx�64�Struct.Union�0�bool
 test.simple�256�0
-tfun�16�(T)�Class�0�auto
+tfun�16�(T v)�Class�0�auto
```

Comment:
not sure if it's better or worse. Source line is
```d
        auto tfun(T)(T v)
        {
                return v;
        }
```

Reply:
The source is valid D code according to GDC.  And from what I can guess, the signature is more likely to be `(T v)` than `(T)` (as `v` is used as a variable in the body); I'd guess the `(T)` part is similar to C++'s `<T>` in `tfun<T>(T v)`.

So I'd say this change is even better.